src/engagement_db_coda_sync/configuration.py

```diff
@@ -16,7 +16,7 @@ class CodaDatasetConfiguration:
     coda_dataset_id: str
     engagement_db_dataset: str
     code_scheme_configurations: [CodeSchemeConfiguration]
-    ws_code_string_value: str
+    ws_code_match_values: [str]
```

Comment:
Why does this need to be an array rather than just a str?
I think since the code scheme's match values are a list, we can always add whatever we need to there to match a single string here, but correct me if I am mistaken.

Reply:
I thought we could have here a list of match values that includes string values (then I'd rename the variable) so that we keep checking for string value and sometimes match values because match values are optional but I've rechecked the ws schemes for consistency and I agree strings will work fine here, thanks.